### PR TITLE
AICORE-436: Wrap search api to use correct audit index name in ES

### PR DESCRIPTION
**Pull request by vpasquier**

### CODE REVIEW

File: nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features({ RestServerFeature.class, EnrichmentTestFeature.class, PlatformFeature.class,
+        RepositoryElasticSearchFeature.class })
+@Deploy({ "org.nuxeo.ai.ai-core", "org.nuxeo.ai.ai-config", "org.nuxeo.ai.ai-model" })
+@Deploy({ "org.nuxeo.ai.ai-config.test:test-es-contrib.xml", "org.nuxeo.elasticsearch.http.readonly" })
+public class TestAISearchREST extends BaseTest {
+
+    protected String payload = "{\n" + "    \"query\": {\n" + "        \"bool\": {\n" + "            \"must\": [\n"
```
Comment (jcarsique) [inline nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java:38]: rather put this in a json resource file

File: nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ * Notice of copyright on this source code does not indicate publication. *
+ *
+ *
+ * Contributors:
+ *     Nuxeo
+ */
+
+package org.nuxeo.ai;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
```
Comment (jcarsique) [inline nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java:14]: shouldn't it be org.assertj.core.api.Assertions.assertThat, not Java6Assertions ?
you could use the common assertEquals here :)

**Review by jcarsique — APPROVED** [review]

File: nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java (hunk 2 above)
Comment (vpasquier) [reply nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java:14]: Oh youre right thank you! Im not a big fan of assertEquals and we did that across the projects already.

File: nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java (hunk 1 above)
Comment (vpasquier) [reply nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java:38]: I think im gonna do a small one without so much stuff

**Pull request merged**